test_requirements/django-4.txt

```diff
@@ -0,0 +1,4 @@
+-r requirements_base.txt
+Django==4.0.1
```

Comment:
I'd probably do this as `>=4.0.1,<4.1` just so we're always testing against the latest version.